src/config.ts

```diff
@@ -18,6 +18,7 @@ export default {
     'yessir',
     'chad',
     'simp',
+    'bitch',
```

Comment:
This is already covered by the library we use https://github.com/retextjs/retext-profanities/blob/main/rules.md

Reply:
what does this mean, I mean already covered in what sense as ("bitch") word was not written on preventing words list

Reply:
oh i got it as it was already written in your profanities rules